src/common/synapse_contracts.ts

```diff
@@ -6,19 +6,29 @@ export namespace SynapseContracts {
     interface SynapseContractArgs {
         bridge:     string;
         bridgeZap?: string;
+        swap?:      string;
     }
 
     export class SynapseContract {
         readonly bridgeAddress:      string;
         readonly bridgeZapAddress?:  string;
+        readonly swapAddress?:       string;
```

Comment:
does this work for the purpose of having a way to get the contract address of the swap contract? it can be manually added to each chain like the bridgeAddress and bridgeZapAddress' are

Reply:
lgtm